config/_default/params.toml

```diff
@@ -16,27 +16,27 @@ domainTLD = "wait4x.dev"
 titleHome = "Wait4X"
 
 ## Twitter Cards
-twitterSite = "@atkrad"
-twitterCreator = "@atkrad"
+twitterSite = "@wait4x"
+twitterCreator = "@wait4x"
 
 ## JSON-LD
 schemaType = "Person"
 # schemaType = "Organization"
 schemaName = "Wait4X"
 schemaAuthor = "Mohammad Abdolirad"
-schemaAuthorTwitter = "https://twitter.com/atkrad"
-schemaAuthorLinkedIn = "https://www.linkedin.com/in/atkrad/"
-schemaAuthorGitHub = "https://github.com/atkrad"
+schemaAuthorTwitter = "https://twitter.com/wait4x"
+schemaAuthorLinkedIn = "https://www.linkedin.com/in/wait4x/"
```

Comment:
We don't have Twitter (X) or LinkedIn account for Wait4X, please return it.

Reply:
Should we remove them?

Reply:
Yes, we can.

Reply:
Done